types/hooks/screens/useSignedChannelListScreenHook.types.ts

```diff
@@ -1,11 +1,12 @@
 import {ChannelList} from '../../database/schema/ChannelList.types';
+import {ContactScreenPayload} from './useChatUtilsHook.types';
 
 interface UseSignedChannelListScreenHook {
   channels: ChannelList[];
   goToChatScreen: (channel: ChannelList) => void;
   goToPostDetailScreen: (channel: ChannelList) => void;
   goToCommunityScreen: (channel: ChannelList) => void;
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function signature for `goToContactScreen` has been updated to accept a parameter of type `ContactScreenPayload`. This is a breaking change if there are any existing usages of this function that do not provide this parameter. Please ensure all usages of this function have been updated accordingly.
```diff
-  goToContactScreen: () => void;
+  goToContactScreen: ({from}: ContactScreenPayload) => void;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->